package.json

```diff
@@ -79,7 +79,7 @@
       "json",
       "ts"
     ],
-    "rootDir": "src",
+    "rootDir": "test",
```

Comment:
Wil je de rootDir weer terugzetten naar `src`? Met jest dienen de tests in dezelfde dir te staan als de source code om coverage te kunnen berekenen. Die mogelijkheid verliezen we in dit geval, maar die zou ik graag willen behouden 😄

Reply:
Goed punt, dacht daar vanochtend zelf ook aan, heb het terug geplaatst.